src/Components/DisplayBookList.js

```diff
@@ -1,9 +1,18 @@
-import React from 'react';
-import { useSelector } from 'react-redux';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 import Book from './Book';
+import { gettingBooks } from '../redux/fetch/booksFetcher';
 
 const DisplayBookList = () => {
-  const books = useSelector((state) => state.books.books);
+  // const books = useSelector((state) => state.books.books);
```

Comment:
- [ Required ] Please consider removing any commented code from your code base, commented code makes your project unprofessional. It would be better if any code that is not necessary is removed from your code base.